packages/dev/buildTools/src/generateDeclaration.ts

```diff
@@ -3,7 +3,7 @@ import * as fs from "fs";
 import * as path from "path";
 import * as chokidar from "chokidar";
 
-import { camelize, checkArgs, checkDirectorySync, debounce, findRootDirectory, getHashOfContent, getHashOfFile } from "./utils";
+import { camelize, checkArgs, checkDirectorySync, debounce, findRootDirectory, getHashOfContent, getHashOfFile, kebabize } from "./utils";

```

Comment:
And the nominees for the best variable name are... :)

Reply:
make it kebab case FTW! :-)

Reply:
This - https://stackoverflow.com/questions/63116039/camelcase-to-kebab-case ;-)